coins/ethereum/examples/eth_quickstart_guide.rs

```diff
@@ -1,25 +1,71 @@
 use walletd_ethereum::prelude::*;
 use walletd_hd_key::prelude::*;
+use ethers::prelude::*;
 
+
+//const PROVIDER_URL: &str = "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161";
+const PROVIDER_URL: &str = "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161";
```

Comment:
The naming is not consistent across the examples. the next example uses INFURA_GOERLI_ENDPOINT instead of PROVIDER_URL